docs/building-a-form/creating-custom-fields-and-widgets.md

```diff
@@ -0,0 +1,58 @@
+# Creating custom fields and widgets
+
+### In this guide
+
+- [How the us-forms-system uses rjsf](#how-us-forms-system-uses-rjsf)
+  - [Customizing fields and widgets from rjsf](customizing-fields-and-widgets-from-rjsf)
+  - [Supporting multi-page forms](#supporting-multi-page-forms)
+
+### How us-forms-system uses rjsf
+
+The us-forms-system code uses rjsf to render form fields, but it builds a scaffolding on top of it to support multi-page forms and our common form patterns. The second part is creating a form configuration spec that allows devs to specify the structure of one of our multi-page forms.
```

Comment:
The "second part" of what? Maybe what this is trying to say is: "Additionally, us-forms-system uses rjsf to create a form configuration spec..."?